applications/IgaApplication/custom_conditions/support_laplacian_condition.cpp

```diff
@@ -20,6 +20,30 @@
 
 namespace Kratos
 {
+    void SupportLaplacianCondition::CalculateLocalSystem(

```

Comment:
LHS and RHS implementations are missing.

Reply:
Sorry @rubenzorrilla but I think this is an outdated support_laplacian_condition. 
Check the last commit I did yesterday